qt/scientific_interfaces/Indirect/IndirectFitDataPresenter.cpp

```diff
@@ -14,14 +14,14 @@ namespace CustomInterfaces {
 namespace IDA {
 
 IndirectFitDataPresenter::IndirectFitDataPresenter(IndirectFittingModel *model,
-                                                   IndirectFitDataView *view)
+                                                   IIndirectFitDataView *view)
```

Comment:
I created an interface class for the `IndirectFitDataView` (called `IIndirectFitDataView`), so that an extra level of indirection can be avoided by not having to put virtual in front of the member functions in the original `IndirectFitDataView`